Cargo.toml

```diff
@@ -49,3 +49,7 @@ pretty_assertions = { default-features = false, version = "1.4.1", features = ["
 [profile.release.package.hello-wasm]
 # Tell `rustc` to optimize for small code size.
 opt-level = "s"
+
+[workspace.lints]
+# By setting the priority to -1, this lint group can be overridden for specific lints
+clippy.all = { level = "warn", priority = -1 }
```

Comment:
What happened with `-Dunused_imports` ?!
I see no entry for it at https://rust-lang.github.io/rust-clippy/master/index.html but Google returns results for such lint ...

Reply:
That's a rustc lint. And it's warn by default, so -Dwarnings will also make it an error